navigation/MainTabNavigator.tsx

```diff
@@ -24,11 +24,14 @@ import { Theme } from '../Theme.style';
 import { StyleSheet } from 'react-native';
 import MediaContext from '../contexts/MediaContext';
 import { GetVideoByVideoTypeQuery } from '../services/API';
+import LiveStreamScreen from '../screens/LiveStreamScreen';
 
 export type HomeStackParamList = {
   HomeScreen: undefined;
   EventDetailsScreen: { item: any };
   AnnouncementDetailsScreen: { item: any };
+  NotesScreen: { date: string }
```

Comment:
This looks incorrect . NotesScreen does not exist in the Home stack.

Reply:
Thanks for the feedback.